CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0] - 2021-07-17
+
+## Added
+
+- Mix task xcribe.doc
```

Comment:
Are we gonna recommend the mix task as the "default" way to generate the docs?

Reply:
Yes @thiamsantos. I just updated the usage instructions at README.